src/components/Result.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+
+export default function Result(props) {
+  return (
+    <p className="col-4" id="result">
+      {props.result}
+    </p>
+  );
+}
```

Comment:
- [ ] For this functional component, use ES6 syntax (arrow function) instead of the function keyword.